AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DatabaseErrorDialog.kt

```diff
@@ -84,6 +84,8 @@ import java.io.IOException
 class DatabaseErrorDialog : AsyncDialogFragment() {
     private lateinit var backups: Array<File>
 
+    private fun requireDeckPicker() = activity as DeckPicker
```

Comment:
Separates the two types of exceptions which can occur

```suggestion
    private fun requireDeckPicker() = requireActivity() as DeckPicker
```